examples/js/loaders/XfileLoader.js

```diff
@@ -0,0 +1,1518 @@
+                    var keys = Object.keys(this.loadingXdata.FrameInfo_Raw);
+                    for (var m = 0; m < keys.length; m++) {
+
+                        if (this.loadingXdata.FrameInfo_Raw[keys[m]].ParentName === this.nowFrameName) {
+
+                            this.loadingXdata.FrameInfo_Raw[this.nowFrameName].children.push(keys[m]);
+
+                        }
+
+                    }
+                    this.frameHierarchie.pop();
+
+                }
+                this.MakeOutputGeometry(this.nowFrameName, this.zflg);
+                this.frameStartLv = this.loadingXdata.FrameInfo_Raw[this.nowFrameName].FrameStartLv;
+                if (this.frameHierarchie.length > 0) {
+
+                    this.nowFrameName = this.frameHierarchie[this.frameHierarchie.length - 1];
+                    this.frameStartLv = this.loadingXdata.FrameInfo_Raw[this.nowFrameName].FrameStartLv;
+
+                } else {
+
+                    this.nowFrameName = "";
+
+                }
+
+            }
+            if (this.nowReadMode === XfileLoadMode$1.Mat_Set) {
+
+                this.loadingXdata.FrameInfo_Raw[this.nowFrameName].Materials.push(this.nowMat);
+                this.nowReadMode = XfileLoadMode$1.Element;
+
+            }
+            this.elementLv--;
+
+        }
+    }, {
+        key: 'beginFrame',
+        value: function beginFrame(line) {
+
+            this.frameStartLv = this.elementLv;
+            this.nowReadMode = XfileLoadMode$1.Element;
+            this.nowFrameName = line.substr(6, line.length - 8);
+            this.loadingXdata.FrameInfo_Raw[this.nowFrameName] = new XFrameInfo$1();
+            this.loadingXdata.FrameInfo_Raw[this.nowFrameName].FrameName = this.nowFrameName;
+            if (this.frameHierarchie.length > 0) {
+
+                this.loadingXdata.FrameInfo_Raw[this.nowFrameName].ParentName = this.frameHierarchie[this.frameHierarchie.length - 1];
+
+            }
+            this.frameHierarchie.push(this.nowFrameName);
+            this.loadingXdata.FrameInfo_Raw[this.nowFrameName].FrameStartLv = this.frameStartLv;
+
+        }
+    }, {
+        key: 'beginReadMesh',
+        value: function beginReadMesh(line) {
+
+            if (this.nowFrameName === "") {
+
+                this.frameStartLv = this.elementLv;
+                this.nowFrameName = line.substr(5, line.length - 6);
+                if (this.nowFrameName === "") {
+
+                    this.nowFrameName = "mesh_" + this.loadingXdata.FrameInfo_Raw.length;
+
+                }
+                this.loadingXdata.FrameInfo_Raw[this.nowFrameName] = new XFrameInfo$1();
+                this.loadingXdata.FrameInfo_Raw[this.nowFrameName].FrameName = this.nowFrameName;
+                this.loadingXdata.FrameInfo_Raw[this.nowFrameName].FrameStartLv = this.frameStartLv;
+
+            }
+            this.loadingXdata.FrameInfo_Raw[this.nowFrameName].Geometry = new THREE.Geometry();
```

Comment:
I'll fixed..but.. create (old) geometry -> BufferGeometry.fromGeometry.
I understand that computational efficiency is bad.